src/get-started/fwe/networking.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Networking and data
+description: Learn how to network your Flutter app.
+prev:
+  title: Handling user input
+next:
+    title: Local data and caching
+---
+
+While it is said that no man is an island, a Flutter app without any networking capability can feel a tad disconnected.
+In this section you will learn how to add networking features to your Flutter app.
+Your app will retrieve data, parse JSON into usable in memory representations, and then send data out again.
+
+## Introduction to retrieving data over the network
+
+The following two tutorials introduce you to the [`http`][] package which enables your app to make [HTTP][] requests with ease, whether you are running on Android, iOS, inside a web browser, or natively on Windows, macOS or Linux. 
+The first tutorial shows you how to make an unauthenticated `GET` request to a website, parse the retrieved data as `JSON` and then display the resulting data.
+The second tutorial builds on the first by adding authentication headers, enabling access to web servers requiring authorization. The article by the Mozilla Developer Network (MDN) gives more background on how authorisation works on the web.
```

Comment:
OMG, you hate line breaks! Also, our docs use American spelling, not Ozzie spelling.  :D

A question. Do these tutorials work for mobile apps when accessing websites?

```suggestion
The following two tutorials introduce you to the
[`http`][] package, which enables your app to make
[HTTP][] requests with ease, whether you are running on Android,
iOS, inside a web browser, or natively on Windows, macOS, or Linux. 
The first tutorial shows you how to make an unauthenticated `GET` request
to a website, parse the retrieved data as `JSON` and then display the resulting data.
The second tutorial builds on the first by adding authentication headers,
enabling access to web servers requiring authorization.
The article by the Mozilla Developer Network (MDN)
gives more background on how authorization works on the web.
```

Reply:
Yes, I'm Australian. Yes, I love run on sentences. I'm trying to get better, but it still slips through =)

As to whether these tutorials are useful from a mobile, I'd have to ask the question as to who is running Xcode and their Flutter tooling on a phone...